product-releases/2021-04-28-falco-support.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Support for Falco
+author: Daniel McSheehy
+author_title: Software Engineer
+author_url: https://github.com/DanielMcSheehy
+author_image_url: https://avatars.githubusercontent.com/u/28493438?s=460&u=4bbd1de69b9ca85d21e45350d15db419a27072fc&v=4
+tags: [security]
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Falco is a popular open-source security and compliance tool.
```

Comment:
why drop the first line? I like that announcement type of line.

Reply:
ah, misunderstood. added back